config.tutorial.yaml

```diff
@@ -1,351 +1,229 @@
-# SPDX-FileCopyrightText: : 2021 The PyPSA-Africa Authors
+library(rasterVis)
```

Comment:
@giacfalk seems the config.tutorial.yaml changed by accident?

Reply:
Yes, something happened by mistake. Sorry about that. Fixing it now.